pyproject.toml

```diff
@@ -11,31 +16,26 @@ keywords= ["Heat Conduction", "FTCS", "Upwind-15", "Crank-Nicholson"]
 exclude = ["test"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-click = "^8.0.1"
-numpy = "^1.24.3"
-importlib-metadata = {version = "^4.8.2", python = "<3.8"}
+python = ">=3.9"
+click = ">=8.0.1"
+numpy = ">=1.24.3"
 
 [tool.poetry.dev-dependencies]
-pytest = "^6.2.5"
+pytest = ">=6.2.5"
 
 [tool.poetry.scripts]
 heateq-design = "heateq_design.__main__:main"
 
-[build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
-
-[tool.coverage.paths]
-source = ["src", "*/site-packages"]
-
 [tool.poetry.metadata]
 classifiers = [
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
     "Development Status :: 3 - Alpha",
     "Intended Audience :: Science/Research",
-    "Topic :: Scientific/Engineering :: Training",
```

Comment:
This is not a valid classifier. See https://pypi.org/classifiers/ - classifiers must come from that list.